exampleCourse/questions/numberExamples/info.json

```diff
@@ -0,0 +1,7 @@
+{
+    "uuid": "933849d9-d7ee-4462-b743-b50e43b03b9e",
+    "title": "Examples Showing Real and Imaginary Input via pl_number_input",
```

Comment:
We are using sentence case (leading capital only) for question titles.

Also, I think it's more accurate to say "real or complex". Maybe we should also drop the "Examples showing" and just say "Real or complex number input via pl-number-input"?